test/instance/00-assert.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 10
+---
+apiVersion: kubernetes.crossplane.io/v1alpha1
+kind: Object
+metadata:
+  annotations:
+    kuttl: '00-install'
+  labels:
+    crossplane.io/claim-name: bucket
+    crossplane.io/claim-namespace: default
+spec:
+  forProvider:
+    manifest:
+      apiVersion: cloudscale.s3.appcat.vshn.io/v1
+      kind: ObjectsUser
+      metadata:
+        name: my-provider-test-bucket-from-claim
+        namespace: default
+  managementPolicy: Default
+  providerConfigRef:
+    name: default
+status:
```

Comment:
I would either only assert the `spec.forProvider` part or the `status.atProvider` part, but not both. It could make sense if patching from references is used, because the result of the reference patches would only be visible in the `status.atProvider` field. But this is not the case here.

Reply:
Went with `spec.forProvider`